_data/contributors.yml

```diff
@@ -400,3 +400,23 @@
   homepage: https://kate-travers.com/
   contributions:
     - blog
+
+- github: zillou
```

Comment:
Thank you for doing this ❤️ 

Could you remove `intro`, `homepage`, and `twitter` as long as they are empty? I believe that will cause some issues.

Reply:
It seems there are many other contributor section also have blank field as well, should I remove them all?

Reply:
Apparently we handle this in the templates, thanks for pointing that out @kenspirit 😁